region/norfair/west/Ice Beam Snake Room.json

```
@@ -440,6 +544,60 @@
         "Alternatively, use a Super just after the Sovas go off screen to speed it up to 40 seconds."
       ]
     },
+    {
+      "link": [2, 2],
+      "name": "G-Mode Setup - Get Hit By Sova",
+      "entranceCondition": {
+        "comeInNormally": {}
+      },
+      "requires": [
+        {"or": [
+          {"and": [
+            {"heatFrames": 600},
```

```suggestion
            {"heatFrames": 2340},
```
And my cue to shoot the super is when the 3rd sova begins going off camera.